### PR TITLE
Allow elastic pool name (instead of id) in sql db update

### LINKED ISSUE #8989: SQL DB UPDATE command does not accept short name for elastic-pool argument
**Describe the bug**
SQL DB UPDATE command does not accept short name for elastic-pool argument

[Documentation](https://docs.microsoft.com/en-us/cli/azure/sql/db?view=azure-cli-latest#az-sql-db-update) for **az-sql-db-update** on elastic-pool argument states: The name or resource id of the elastic pool to create the database in.

**This does not work:**
`user@machine:~$ az sql db update -g resgrp-s resgrp-sqlsrv -n madb_Copy --elastic-pool mypool`
Invalid resource identifier 'mypool'.

**This works:**
`user@machine:~$ az sql db update -g resgrp-s resgrp-sqlsrv -n madb_Copy --elastic-pool /subscriptions/xxxx-xxxx-xxxx/resourceGroups/rgjg/providers/Microsoft.Sql/servers/rgjg/elasticPools/mypool`

**To Reproduce**
Perform commands above.

**Expected behavior**
As defined by documentation.

**Environment summary**
Bash on Windows
azure-cli v2.0.61
python 3.6.5
_same in powershell on Windows 10_

### PR DESCRIPTION
**Pull request by jaredmoo**
Fixes #8989

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-sql/HISTORY.rst [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 Release History
 ===============
+
 2.2.1
 +++++
 * Updating az sql mi create and update with two new properties (proxyOverride and publicDataEndpointEnabled)
+* Allow elastic pool name (instead of id) to be specified for az sql db update.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-sql/HISTORY.rst]: Just to be certain, the corrected behavior will allow elastic pool name or ID, correct?

Comment (jaredmoo) [reply src/command_modules/azure-cli-sql/HISTORY.rst]: That's right

Comment (jaredmoo) [reply src/command_modules/azure-cli-sql/HISTORY.rst]: Let me double check that there is test coverage of both

Comment (jaredmoo) [reply src/command_modules/azure-cli-sql/HISTORY.rst]: Updated test and tried to clarify history entry

**Pull request merged**